File: C/Estrutura repetitiva/divisao.c
// Biblioteca
#include <stdio.h>

int main()
{
    // Variável
    int quant, i, num, den;
    double divisao;

    // Entrada de dados
    printf("Digite quantos casos voce vai digitar: ");
    scanf("%d", &quant);

    // Estrutura repetitiva
    for (i=1; i <= quant ; i++) {
        // Entrada de dados
        printf("Entre com o numerador: ");
        scanf("%d", &num);
        printf("Entre com o denominador: ");
        scanf("%d", &den);

        // Estrutura condicional
        if (den == 0) {
            printf("Divisao impossivel!\n");
        }
        else {
            // Algoritmo matemático
            divisao = (double) num / den;
            printf("A divisao entre %d / %d e %.2lf\n", num, den, divisao);
        }
    }

    return 0;
}

File: C#/Estrutura repetitiva/divisao.cs
// Variável
int quant, i, num, den;
double divisao;

// Entrada de dados
Console.Write("Digite quantos casos você vai digitar: ");
quant = int.Parse(Console.ReadLine());

// Estrutura repetitiva
for (i = 1; i <= quant; i++)
{
    // Entrada de dados
    Console.Write("Entre com o numerador: ");
    num = int.Parse(Console.ReadLine());

    Console.Write("Entre com o denominador: ");
    den = int.Parse(Console.ReadLine());

    // Estrutura condicional
    if (den == 0)
    {
        Console.WriteLine("Divisão impossível!");
    }
    else
    {
        // Algoritmo matemático
        divisao = (double) num / den;
        Console.WriteLine("A divisão entre " + num + " / " + den + " é " + divisao.ToString("F2"));
    }
}